src/scanForInputs.js

```diff
@@ -38,85 +102,87 @@ const scanForInputs = (DeviceInterface, forms = _forms) => {
         return input
     }
 
-    const addInput = (input) => {
-        const parentForm = getParentForm(input)
+    /**
+     * @param {HTMLInputElement|HTMLSelectElement} input
+     */
+    addInput (input) {
+        const parentForm = this.getParentForm(input)
 
         // Note that el.contains returns true for el itself
-        const previouslyFoundParent = [...forms.keys()].find((form) => form.contains(parentForm))
+        const previouslyFoundParent = [...this.forms.keys()].find((form) => form.contains(parentForm))
 
         if (previouslyFoundParent) {
             // If we've already met the form or a descendant, add the input
-            forms.get(previouslyFoundParent)?.addInput(input)
+            this.forms.get(previouslyFoundParent)?.addInput(input)
         } else {
             // if this form is an ancestor of an existing form, remove that before adding this
-            const childForm = [...forms.keys()].find((form) => parentForm.contains(form))
+            const childForm = [...this.forms.keys()].find((form) => parentForm.contains(form))
             if (childForm) {
-                forms.get(childForm)?.destroy()
-                forms.delete(childForm)
+                this.forms.get(childForm)?.destroy()
+                this.forms.delete(childForm)
             }
 
-            forms.set(parentForm, new Form(parentForm, input, DeviceInterface))
+            this.forms.set(parentForm, new Form(parentForm, input, this.device))
         }
     }
 
-    const findEligibleInputs = (context) => {
-        if (context.matches?.(FORM_INPUTS_SELECTOR)) {
-            addInput(context)
-        } else {
-            context.querySelectorAll(FORM_INPUTS_SELECTOR).forEach(addInput)
+    /**
+     * enqueue elements to be re-scanned after the given
+     * amount of time has elapsed.
+     *
+     * @param {(HTMLElement|Document)[]} htmlElements
+     */
+    enqueue (htmlElements) {
+        for (let element of htmlElements) {
+            this.changedElements.add(element)
         }
+        clearTimeout(this.debounceTimer)
+        this.debounceTimer = setTimeout(() => {
+            this.processChangedElements()
+        }, this.debounceTimePeriod)
```

Comment:
Does this apply to all platforms now, though? I think we were using `requestIdleCallback` which should be preferable when available, right?